volume-providers/vpc/provider/util.go

```diff
@@ -41,9 +42,19 @@ var skipErrorCodes = map[string]bool{
 	"invalid_route":                    false,
 }
 
+func getEnv(key string) string {
+	return os.Getenv(strings.ToUpper(key))
+}
+
 // retry ...
 func retry(logger *zap.Logger, retryfunc func() error) error {
 	var err error
+	if vpcRetryInterval := getEnv("VPC_RETRY_INTERVAL"); vpcRetryInterval != "" {
+		maxRetryGap, _ = strconv.Atoi(vpcRetryInterval)
+	}
+	if vpcRetryAttempt := getEnv("VPC_RETRY_ATTEMPT"); vpcRetryAttempt != "" {
+		maxRetryAttempt, _ = strconv.Atoi(vpcRetryAttempt)
```

Comment:
looks maxRetryAttempt can be unexpected in case of `strconv.Atoi` fail, please check